docs/conf.py

```diff
@@ -265,5 +265,6 @@
 intersphinx_mapping['regions'] = ('https://astropy-regions.readthedocs.io/en/stable/', None)
 intersphinx_mapping['skimage'] = ('https://scikit-image.org/docs/stable/', None)
 intersphinx_mapping['specutils'] = ('https://specutils.readthedocs.io/en/stable/', None)
+intersphinx_mapping['specreduce'] = ('https://specreduce.readthedocs.io/en/latest/', None)
```

Comment:
Is the stable unreliable? Is there a reason to prefer latest?

Reply:
both do exist, I just used latest since that is the default redirect from specreduce.readthedocs.io, but have no objections to switching to stable if you think it should be that instead?

Reply:
I think stable will be less confusing to the user, since our pip install pulls in stable version of specreduce, not the dev version.

Reply:
```suggestion
intersphinx_mapping['specreduce'] = ('https://specreduce.readthedocs.io/en/stable/', None)
```

Reply:
ah that's right, the API isn't there and is causing the build to fail.  Would you prefer I revert this back to latest or remove the mapping entirely for now?

Reply:
Wait... we're using an unreleased API?

Reply:
Oh... I see. They not rendering it at all. Hmm. latest is fine for now, but maybe open follow up issue to use stable again when they have caught up. Thx!

Reply:
since we don't link to any of the API yet, should I just remove the mapping?  That might be easier to remember to update than a follow-up issue?  Or is there another reason it should be in the mapping since its already a dependency?

Reply:
If it is not used in Sphinx ref, then you can remove it. This part is only for doc.